README.md

```diff
@@ -49,6 +49,8 @@ You can install this plugin using the [Jetbrains plugin repository](https://plug
 - Install latest versions of [Hindent](https://github.com/chrisdone/hindent) and [Stylish-Haskell](https://github.com/jaspervdj/stylish-haskell). 
     Note that currently the latest version of Hindent is not on Stackage LTS. You have to install version of Hindent > 5.0, for example by: `stack install --resolver nightly-2016-11-06 hindent`
     Set file paths to `hindent` and `stylish-haskell` in the `Settings`/`Other Settings`/`Haskell`;
+- Install latest version of [haskell-tools](https://github.com/haskell-tools/haskell-tools) from stackage, e.g. `stack install --resolver nightly-2017-01-14 haskell-tools-cli`.
```

Comment:
> But all packages all have it start point resolver in stackage right? I mean if a new package is released in stackage, it will only be compatible in older resolvers and there will no promise for the previews resolver compatibility.

I do not understand you. 

My point is that tools like `ghc-mod`, `intero` and also `haskell-tools` are depending on specific version of GHC API. So, if your project is build with certain GHC version, these tools also have to be build by same GHC version.